packages/addons/browser/chromium/source/resources/settings.xml

```diff
@@ -10,6 +10,7 @@
         <setting id="HOMEPAGE" type="text" label="1025" default="https://libreelec.tv" />
         <setting id="USE_CUST_AUDIODEVICE" type="bool" label="1026" default="false" />
         <setting id="CUST_AUDIODEVICE_STR" type="text" label="1027" visible="eq(-1,true)" subsetting="true" default="" />
+        <setting id="UNCLUTTER" type="bool" label="1028" default="false" />
```

Comment:
If the add-on setting is "Hide Cursor" perhaps `"HIDE_CURSOR_UNCLUTTER"` would be a better name so there's a clear connection between the setting and the `if [ "$UNCLUTTER" == "true" ]` statements?